tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -947,8 +947,8 @@ class ExecutionTest {
         D3D12_RESOURCE_STATE_COPY_DEST, nullptr, IID_PPV_ARGS(&pReadBuffer)));
 
     transferData.pData = values;
-    transferData.RowPitch = valueSizeInBytes/resDesc.Height;
-    transferData.SlicePitch = valueSizeInBytes;
+    transferData.RowPitch = (LONG_PTR)(valueSizeInBytes/resDesc.Height);
```

Comment:
Using LONG_PTR for a non-pointer value seems a bit strange. If we need a pointer-sized arithmetic type, then SIZE_T would make more sense here I think.

Reply:
I used LONG_PTR here because it is the type of the [D3D12_SUBRESOURCE_DATA ](https://docs.microsoft.com/en-us/windows/win32/api/d3d12/ns-d3d12-d3d12_subresource_data) structure RowPitch and SlicePitch members.

Reply:
Well that makes it a very good choice indeed here :)